packages/ember-views/lib/attr_nodes/attr_node.js

```diff
@@ -15,6 +15,12 @@ export default function AttrNode(attrName, attrValue) {
   this.init(attrName, attrValue);
 }
 
+export var styleWarning = 'Dynamic content in the `style` attribute is not escaped and may expose your application ' +
```

Comment:
this seems pretty verbose, I fear heavy signal-to-noise. I would be a fan of a single brief sentence and a link for more info.

> unsafe use of style attribute bindings, please refer to: ....